icons/layers-2.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M22 8c0-.4-.3-.8-.8-1.1l-7.6-4.5a2.93 2.93 0 0 0-3.1 0L2.8 6.9c-.5.3-.8.7-.8 1.1s.3.8.8 1.1l7.6 4.5a3.3 3.3 0 0 0 3 0L21 9.1c.7-.3 1-.7 1-1.1" />
+  <path d="m7.7 12-4.9 2.9c-.5.3-.8.7-.8 1.1s.3.8.8 1.1l7.6 4.5a3.3 3.3 0 0 0 3 0l7.6-4.5c.5-.3.8-.7.8-1.1s-.3-.8-.8-1.1L16.3 12" />
```

Comment:
What about calling this `two-layers`?

IMO we should just make it a hard requirement to come up with a good name before we merge even the cutest of icons.

Reply:
> IMO we should just make it a hard requirement to come up with a good name before we merge even the cutest of icons.

@jguddas Agree in general we should strive for better naming, but not sure about how _hard_ the requirement should be… In some cases—and specifically in this case—`-2` & `-3` actually work well as coincidentally matching the number of layers. cc: @karsa-mistmere @ericfennis 

> What about calling this `two-layers`?

I don’t see how `two` is preferable to `2` here… Typing `two` instead of `2` in search is less user friendly.

Reply:
The corners are a bit wonky:
![image](https://github.com/lucide-icons/lucide/assets/17746067/ba819f4c-98ff-44f5-9007-938b807a65f3)

```suggestion
  <path d="m16.02 12 5.48 3.13a1 1 0 0 1 0 1.74L13 21.74a2 2 0 0 1-2 0l-8.5-4.87a1 1 0 0 1 0-1.74L7.98 12" />
  <path d="M13 13.74a2 2 0 0 1-2 0L2.5 8.87a1 1 0 0 1 0-1.74L11 2.26a2 2 0 0 1 2 0l8.5 4.87a1 1 0 0 1 0 1.74Z" />
```